modules/nextMillenniumBidAdapter.js

```diff
@@ -24,12 +25,18 @@ export const spec = {
 
     _each(validBidRequests, function(bid) {
       window.nmmRefreshCounts[bid.adUnitCode] = window.nmmRefreshCounts[bid.adUnitCode] || 0;
+      const id = getPlacementId(bid)
+
+      if (bid.sizes && !Array.isArray(bid.sizes[0])) bid.sizes = [bid.sizes]
+      if (!bid.ortb2) bid.ortb2 = {}
+      if (!bid.ortb2.device) bid.ortb2.device = {}
+      bid.ortb2.device.referrer = (getRefererInfo && getRefererInfo().ref) || ''
```

Comment:
@patmmccann, 
Thank you so much for reviewing and approving this PR,
This is currently live, but is however creating an unexpected error, which I can't figure out why.
I know it's not your job, but I would greatly appreciate it if you can maybe help me out here for a quick second, 

I'll attach a screenshot of the issue, but in short the bid.ortb2 object is empty, and my fallback line for such a case (32) seems to not work (as it's still an empty object in line 33)

![image](https://user-images.githubusercontent.com/42449375/185251971-52724787-1083-4684-9a40-e259bf14f2cd.png)

If you want to see a case where this is happening you can see it on [this](https://tettybetty.com/feel-good-surfing-art-by-jack-soren/?pb-debug=true) page